inst/tests/testthat/test-helloworld.R

```diff
@@ -0,0 +1,34 @@
+library(testthat)
+library(methods)
+library(grpc)
+library(processx)
+
+context('helloworld example')
+
+## start a server in the background
+p <- process$new(commandline = "R -e \"demo('helloserver', 'grpc', ask = FALSE)\"", stdout = "|", stderr = "|")
+
+test_that('server started', {
+    expect_true(p$is_alive())
+})
+
+## wait for the server to start
+while (TRUE) {
```

Comment:
`for(i in 1:10)` instead of infinite loop

Reply:
Yeah, good point, we should give up after a few seconds. So let's do this instead!